### DISCUSSION TITLE
Remove and refetch at the same call

### DISCUSSION THREAD

**Discussion by cubecleveland** [TanStack/query#2858] [Q&A] [answered]
Hello,

Im trying to remove and refetch a query at the same time...
Basically im trying to trigger the isLoading state and refetch the query, like it was initially loading.
I am currnetly calling remove and refetch on the query returnd objetc. is there any way to do it in 1 call ?

**Comment by TkDodo** [comment, marked as answer]
`queryClient.resetQueries` is what you're looking for I think
